auth/postgres/tracing.go

```diff
@@ -30,14 +32,24 @@ func NewDatabase(db *sqlx.DB) Database {
 	}
 }
 
-func (d database) NamedExecContext(ctx context.Context, query string, args interface{}) (sql.Result, error) {
+func (dm database) NamedQueryContext(ctx context.Context, query string, args interface{}) (*sqlx.Rows, error) {
```

Comment:
Why was this changed to `dm`, and what `m` really represents here (a bit confusing)?